Tools-Override/tests.targets

```diff
@@ -211,8 +211,54 @@
       <TestCommandLines Include="cd native"/>
     </ItemGroup>
 
+    <!-- For UAP F5 scenario, generate the test commands -->
+    <Error Condition="'$(BuildingUAPVertical)' == 'true' and '$(UAPRunnerFolder)'==''" Text="Please specify UAPRunnerFolder" />
```

Comment:
+1. We should move this error to the `RunTestsForProject` target instead since that one is the one that really needs it